softlayer/resource_softlayer_scale_group.go

```diff
@@ -334,29 +356,31 @@ func resourceSoftLayerScaleGroupRead(d *schema.ResourceData, meta interface{}) e
 	d.Set("cooldown", *slGroupObj.Cooldown)
 	d.Set("status", *slGroupObj.Status.KeyName)
 	d.Set("termination_policy", *slGroupObj.TerminationPolicy.KeyName)
-	d.Set("virtual_server_id", *slGroupObj.LoadBalancers[0].VirtualServerId)
-	d.Set("port", *slGroupObj.LoadBalancers[0].Port)
-
-	// Health Check
-	healthCheckObj := slGroupObj.LoadBalancers[0].HealthCheck
-	currentHealthCheck := d.Get("health_check").(map[string]interface{})
-
-	currentHealthCheck["type"] = *healthCheckObj.Type.Keyname
-
-	if *healthCheckObj.Type.Keyname == HEALTH_CHECK_TYPE_HTTP_CUSTOM {
-		for _, elem := range healthCheckObj.Attributes {
-			switch *elem.Type.Keyname {
-			case "HTTP_CUSTOM_TYPE":
-				currentHealthCheck["custom_method"] = *elem.Value
-			case "LOCATION":
-				currentHealthCheck["custom_request"] = *elem.Value
-			case "EXPECTED_RESPONSE":
-				currentHealthCheck["custom_response"] = *elem.Value
+	if slGroupObj.LoadBalancers != nil && len(slGroupObj.LoadBalancers) > 0 {
```

Comment:
`.LoadBalancers` is a slice, not a pointer nor interface{}. So I don't think it would ever be nil.